bosh/ruby-app-release/src/ruby-web-app/views/index.erb

```diff
@@ -0,0 +1,10 @@
+<body style="background-color:#dddddd">
+<center>
+<img src="https://www.anynines.com/assets/press/logo-color-on-white-d7fc3d56c43f8b3547fedfd462ff6e44b6cb571032ee67d7c60491a7ab82e1a2.png" />
```

Comment:
Let's try to avoid any mentioning of `anynines` so that it can't be found when using the GitHub search